Survey/Survey.xcodeproj/project.pbxproj

```diff
@@ -80,13 +88,31 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		C958042F2C9F636F000116A0 /* Models */ = {
+			isa = PBXGroup;
+			children = (
+				C9253F832C9F6940003F075D /* SurveyViewModel.swift */,
+				C9253F852C9F6B7E003F075D /* InquiryModel.swift */,
+			);
+			path = Models;
+			sourceTree = "<group>";
+		};
 		C96DB3742C9D0CF900D7001D /* Preview Content */ = {
 			isa = PBXGroup;
 			children = (
 			);
 			path = "Preview Content";
```

Comment:
I'm actually getting a build error building this project saying that this path doesn't exist.

Reply:
Maybe you didn't commit these files?